### DISCUSSION TITLE
Emgucv build from source on Jetson Ubuntu (18.0.4)

### DISCUSSION THREAD

**Discussion by mathur01** [emgucv/emgucv#849] [Q&A]
Hi all,
I am trying to build the source on the Jetson Nano device, which has the Ububtu (18.0.4) version. I followed the latest emugcv version guides for Ubunut 22 version setup instructions, but i faced some dependency issues. 

Then, i also tried to install an older version (emugcv 4.2) to resolve this issue, but the build was not successful. So do we have instructions to install emugcv 4.2 as a git repo? Do we have instructions to build from older versions as zip files?

**Comment by sn4k3** [comment]
For arm I would try the compile script from [Debian](https://github.com/emgucv/emgucv/tree/master/platforms/debian/bullseye)

**Reply by mathur01** [reply]
Hi Thanks for your reply, but it dint worked. Geting failed in build. Looks like it need some sepcified version of dependcenes.
